### DISCUSSION TITLE
View bug when alternating between Altscreen and main screen

### DISCUSSION THREAD

**Discussion by bernardo1r** [charmbracelet/bubbletea#613] [Q&A]
Sometimes when entering and exiting altscreen some text gets trapped in the main screen.
I've already tried many solutions, including using a variable ```quitting```, as in the examples, to output an empty string ```""``` when set, but without any success. An odd solution was to wait for a short time with the ```View()``` outputting nothing before entering the altscreen e.g. ```time.Sleep(time.Millisecond * 100)```, and then when exiting the altscreen, the previous output was not present anymore.
Has anyone encountered this same issue, or am I doing something wrong?

**Comment by meowgorithm** [comment]
Hey, there! Any source you could include or point us towards so we can diagnose accordingly?

**Comment by bashbunni** [comment]
@bernardo1r bump! Do you have any code examples that reproduce the issue so we can try it out on our end?
